test/homework_01/test_task_02.py

```diff
@@ -0,0 +1,42 @@
+from homework_01.task_02 import check_fibonacci
+
+
+def test_positive_case_from_0():
+    """Testing that several consecutive Fibonacci numbers starting with 0 give True"""
+    assert check_fibonacci([0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377])
```

Comment:
I'd say 5 elements is enough,  try to make test data as small as possible.  I'd say 5 or 10 items work pretty the same for that task.